package.json

```diff
@@ -97,6 +97,7 @@
 		"/retire-js",
 		"/oclif.lock"
 	],
+	"types": "/src/types.ts",
```

Comment:
What does this "types" property on the package.json file do? Do we need it?

Reply:
Turned out, we didn't need it. I have removed this.